libs/gi/ui/src/components/database/DatabaseCard.tsx

```diff
@@ -63,7 +63,17 @@ function DataCard({ index, readOnly }: { index: number; readOnly: boolean }) {
   const numChar = database.chars.keys.length
   const numArt = database.arts.values.length
   const numWeapon = database.weapons.values.length
-  const hasData = Boolean(numChar || numArt || numWeapon)
+  const numTeams = database.teams.values.length
+  let numLoadouts = 0;
```

Comment:
The number of loadouts should just be `database.teamChars.values.length`. 

Can you also include the number of builds?
`database.builds.values.length`